### PR TITLE
scene layer select sample

### PR DESCRIPTION
**Pull request by tschie**
shows how to select geoelements in a scene layer (in this case buildings)

### CODE REVIEW

**Comment by tschie** [comment]
@john0005 please review when you have time

File: src/main/java/com/esri/samples/scene/scene_layer_select/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    <li>From the resulting <code>IdentifyLayerResult</code>, get the list of identified <code>GeoElement</code>s with
+     <code>result.getElements()</code>.</li>
+     <li>Get the first element in the list, checking that it is a feature, and call <code>sceneLayer.selectFeature
+     (feature)</code> to select it.</li>
+</ol>
+
+<h2>Features</h2> 
+
+<ul>
+    <li>ArcGISSceneLayer</li>
```
Comment (john0005) [inline src/main/java/com/esri/samples/scene/scene_layer_select/README.md:29]: Would probably add GeoElement here as well.

File: src/main/java/com/esri/samples/scene/scene_layer_select/SceneLayerSelectSample.java [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+                      // select the feature
+                      sceneLayer.selectFeature((Feature) geoElement);
+                    }
+                  }
+                } catch (InterruptedException | ExecutionException ex) {
+                  new Alert(Alert.AlertType.ERROR, "Error identifying features").show();
+                }
+              });
+            } else {
+              new Alert(Alert.AlertType.ERROR, "Error loading scene layer").show();
```
Comment (john0005) [inline src/main/java/com/esri/samples/scene/scene_layer_select/SceneLayerSelectSample.java]: This error throws for me when I am trying to rotate or drag around the map.

**Comment by john0005** [comment]
Will also need to make an issue for Mac because selecting doesn't seem to show up that well.
screen shot 2018-06-05 at 2 15 47 pm

**Review by john0005 — APPROVED** [review]
Looks good!

**Pull request merged**